nitro-protocol/contracts/MultiAssetHolder.sol

```diff
@@ -643,7 +533,13 @@ contract MultiAssetHolder is IMultiAssetHolder, StatusManager {
         return a > b ? b : a;
     }
 
-    function decodeGuaranteeData(bytes memory data) internal pure returns (bytes32[] memory) {
-        return abi.decode(data, (bytes32[]));
+    struct Guarantee {
+        // TODO absorb into exit format
```

Comment:
Do we want to do some specific work prior to adding this into exit format?

Reply:
Thinking about this some more, I am not sure it belongs in the exit format. So I removed the TODO [b3045f3](https://github.com/statechannels/go-nitro/pull/810/commits/b3045f3a4dbfbc31b3bed8780938955e83b99597).

Reply:
Actually, you are right. Guarantee concept is more nitro-related.